dap_prinz_green_jobs/analysis/ojo_analysis/aggregate_by_region.py

```diff
@@ -0,0 +1,93 @@
+from dap_prinz_green_jobs.analysis.ojo_analysis.process_ojo_green_measures import *
```

Comment:
Could be good to add a quick one liner about what this script does and how to run it